paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -2682,4 +2693,33 @@ public InlineElement getDescription() {
             return text(description);
         }
     }
+
+    /** Time field type for record level expire. */
+    public enum TimeFieldType implements DescribedEnum {
+        SECOND(
+                "second",
```

Comment:
`seconds-int`

Reply:
Fixed in https://github.com/apache/paimon/commit/20a186ed4c7abbf08f9b89ae33f74a5075d068dd